core-it-suite/src/test/java/org/apache/maven/it/MavenITmng6090CIFriendlyTest.java

```diff
@@ -61,7 +60,7 @@ public void testitShouldResolveTheDependencies()
         
         verifier.addCliOption( "-Drevision=1.2" );
         verifier.setLogFileName( "install-log.txt" );
-        verifier.executeGoals( Arrays.asList( "clean", "install" ) );
```

Comment:
Please don't change other integration tests, only the version-range if the behavior changes (or add code)
I'm pretty sure you want to test the `install` phase when working with CI-friendly versions, to ensure that still works.

Reply:
Right, I'm not sure if this was an intended change. I've reverted it.

Reply:
It was an intended change, we figured that `clean install` was no longer needed. But I guess it'll still work 😄